node/src/command.rs

```diff
@@ -194,24 +203,33 @@ macro_rules! construct_sync_run {
 	(|$components:ident, $cli:ident, $cmd:ident, $config:ident| $code:expr ) => {{
 		let runner = $cli.create_runner($cmd)?;
 
+		// none of these are using instant seal
+		let is_standalone = false;
 		match runner.config().chain_spec.identify() {
 			ChainIdentity::Amplitude => runner.sync_run(|$config| {
 				let $components = new_partial::<
 					amplitude_runtime::RuntimeApi,
 					AmplitudeRuntimeExecutor,
-				>(&$config)?;
+				>(&$config, is_standalone)?;
 				$code
 			}),
 			ChainIdentity::Foucoco => runner.sync_run(|$config| {
-				let $components =
-					new_partial::<foucoco_runtime::RuntimeApi, FoucocoRuntimeExecutor>(&$config)?;
+				let $components = new_partial::<foucoco_runtime::RuntimeApi, FoucocoRuntimeExecutor>(
+					&$config,
+					is_standalone,
+				)?;
 				$code
 			}),
 			ChainIdentity::Pendulum => runner.sync_run(|$config| {
-				let $components =
-					new_partial::<pendulum_runtime::RuntimeApi, PendulumRuntimeExecutor>(&$config)?;
+				let $components = new_partial::<
+					pendulum_runtime::RuntimeApi,
+					PendulumRuntimeExecutor,
+				>(&$config, is_standalone)?;
 				$code
 			}),
+			// Foucoco standalone is only supported
+			// with the instant seal flag.
+			ChainIdentity::FoucocoStandalone => todo!(),
```

Comment:
In the latter case, I think we should rather mark it as `unreachable!` if logically impossible or as `unimplemented!` if we don't actually intend to implement that case.

Reply:
Right, I forgot about those macros. I believe `unimplemented` makes more sense since it can actually be called with those flags, they just don't make sense for now.